documentation/features/scheduling.rst

```diff
@@ -0,0 +1,223 @@
+   * - ``site-power-capacity``
+     - ``"45kW"``
+     - Maximum/minimum achievable power at the grid connection point [#asymmetric]_. Defaults to the Asset attribute ``capacity_in_mw``
+   * - ``site-consumption-capacity``
+     - ``"45kW"``
+     - Maximum consumption power at the grid connection point [#consumption]_. If ``site-power-capacity`` is defined, the minimum between the ``site-power-capacity`` and ``site-consumption-capacity`` will be used. Defaults to the Asset attribute ``consumption_capacity_in_mw``
+   * - ``site-production-capacity``
+     - ``"0kW"``
+     - Maximum production power at the grid connection point [#production]_.  If ``site-power-capacity`` is defined, the minimum between the ``site-power-capacity`` and ``site-production-capacity`` will be used. Defaults to the Asset attribute ``production_capacity_in_mw``
+
+
+.. [#asymmetric] ``site-consumption-capacity`` and ``site-production-capacity`` allow defining asymmetric contracted transport capacities for each direction (i.e. production and consumption).
+.. [#production] Example: with a connection capacity (``site-power-capacity``) of 1 MVA (apparent power) and a production capacity (``site-production-capacity``) of 400 kW (active power), the scheduler will make sure that the grid outflow doesn't exceed 400 kW.
+.. [#consumption] Example: with a connection capacity (``site-power-capacity``) of 1 MVA (apparent power) and a consumption capacity (``site-consumption-capacity``) of 800 kW (active power), the scheduler will make sure that the grid inflow doesn't exceed 800 kW.
+
+.. warning:: If no (symmetric, consumption and production) site capacity is defined (also not as defaults), the scheduler will not enforce any bound on the site power.
+
+
+The flex-models & corresponding schedulers
+-------------------------------------------
+
+Storage
+^^^^^^^^
+
+For *storage* devices , the FlexMeasures scheduler deals with the state of charge (SoC) for an optimal outcome.
+You can do a lot with this ― examples for storage devices are:
+
+- batteries
+- :abbr:`EV (electric vehicle)` batteries connected to charge points
+- hot water storage ("heat batteries", where the SoC relates to the water temperature)
+- pumped hydro storage (SoC is the water level)
+- water basins (here, SoC is supposed to be low, as water is being pumped out)
+- buffers of energy-intensive chemicals which are needed in other industry processes
+
+
+The ``flex-model`` for storage describes to the scheduler what the flexible asset's state is,
+and what constraints or preferences should be taken into account.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 25 90
+
+   * - Field
+     - Example value
+     - Description 
+   * - ``soc-at-start``
+     - ``"3.1"``
+     - The (estimated) state of charge at the beginning of the schedule (defaults to 0).
+   * - ``soc-unit``
+     - ``"kWh"`` or ``"MWh"``
+     - The unit which the schedule should be in.
```

Comment:
This is incorrect. The unit of the schedule is defined by the unit of the power sensor on which the schedule is stored. The `soc-unit` instead defines the unit for all SoC related flex-model fields.